detection-rules/body_extortion.yml

```diff
@@ -22,8 +24,12 @@ source: |
       sender.email.domain.root_domain not in $free_email_providers
       and sender.email.domain.domain not in $sender_domains
     )
+  
     // many extortion emails spoof sender domains and fail sender authentication
-    or any(headers.hops, .authentication_results.dmarc == "fail")
+    or any(headers.hops,
+           .authentication_results.dmarc == "fail"
+           or .authentication_results.compauth.verdict not in ("pass", "softpass")
```

Comment:
@morriscode what happens if there's no SPF/DMARC configured in this case? we could check the explicit failure conditions instead if we're worried about that